vm/src/builtins/dict.rs

```diff
@@ -946,14 +946,23 @@ trait ViewSetOps: DictView {
         Ok(PySet { inner })
     }
 
-    #[pymethod(name = "__rsub__")]
+    #[pymethod(name = "__sub__")]
```

Comment:
`__sub__` is handled by below `#[pymethod(magic)]`. Remove this line if you are going to add separated rsub